test/optimizer/table_stats_test.cpp

```diff
@@ -0,0 +1,88 @@
+#include <memory>
+
+#include "gtest/gtest.h"
+#include "optimizer/statistics/table_stats.h"
+
+#include "util/test_harness.h"
+
+namespace terrier::optimizer {
+class TableStatsTests : public TerrierTest {
+ public:
+  auto column_stats_obj_1 = ColumnStats(catalog::db_oid_t(1), catalog::table_oid_t(1), catalog::col_oid_t(1), 5, 4, 0.2,
```

Comment:
Initialize these in the `SetUp()` method for the class. See this example:

https://github.com/cmu-db/terrier/blob/master/test/network/network_test.cpp#L45-L64

Make sure you call `TerrierTest::SetUp();` in your method.